api-client/src/main/java/com/xing/api/RealCallSpec.java

```diff
@@ -142,6 +144,16 @@ public void onResponse(Call call, okhttp3.Response rawResponse) {
     }
 
     @Override
+    public Single<RT> singleStream() {
+        return stream().toSingle();
+    }
+
+    @Override
+    public Completable completableStream() {
+        return stream().toCompletable();
```

Comment:
Well the response will be parsed and everything, the only difference is that at the end there's no onNext() and therefore the response is just ignored as if you just call subscribe without arguments.